windows/installer/MozillaVPN_prod.wxs

```diff
@@ -11,81 +11,85 @@
 <?define WintunPlatform = "amd64" ?>
 
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
-	<Product
+  <Product
 		Id="*"
 		Name="Mozilla VPN"
 		Language="1033"
 		Version="!(bind.FileVersion.MozillaVPNExecutable)"
 		Manufacturer="Mozilla Corporation"
 		UpgradeCode="$(var.UpgradeCode)">
-		<Package
+    <Condition Message="This application is only supported on Windows 10 or higher.">
+      <![CDATA[Installed OR (VersionNT >= 603)]]>
+    </Condition>
+    <Condition Message="This application is only supported on 64 bit x86 platforms."> %PROCESSOR_ARCHITECTURE="AMD64" </Condition>
```

Comment:
https://wixtoolset.org/documentation/manual/v3/howtos/redistributables_and_install_checks/block_install_on_os.html
IN here they say to check the `VersionNT64` var. I cannot find the documentation for this `PROCESS_ARCHITECTURE="AMD64"` can you please share some?

Reply:
I used it based on this:
https://stackoverflow.com/questions/8853653/wix-installer-differentiate-32bit-from-64bits
We can't use VersionNT64 because ARM64 would still be 64bit.